troubadix/plugins/valid_oid.py

```diff
@@ -403,6 +403,7 @@ def check_content(
         exceptions = [
             "ossim_server_detect.nasl",
             "gsf/2018/vmware/gb_vmware_fusion_vmxnet3_"
+            "stack_memory_usage_vuln_macosx.nasl",
             "enterprise/2018/vmware/gb_vmware_fusion_vmxnet3_"
```

Comment:
please add on all line ends a `,`

Reply:
Please correct me if i'm wrong but that would mean that `gsf/2018/vmware/gb_vmware_fusion_vmxnet3_` and `stack_memory_usage_vuln_macosx.nasl` would be seen as separate strings / files while it should be `gsf/2018/vmware/gb_vmware_fusion_vmxnet3_stack_memory_usage_vuln_macosx.nasl` (a single file / string).

Reply:
You're right, I just thought they were all different files and not contiguous strings.